### ISSUE TITLE
"Hide sponsored contents" not working 

### ISSUE THREAD

**Issue by SickClang** [ZeeRooo/MaterialFBook#238]
I see that the last activities on this repo are old, but the functionality mentioned above is not working. Even if the switch button is activated, I see the sponsored posts.

**Comment by DickinsonSa** [comment]
Hi, have you tried reinstalling? It fixed an issue for me.

**Comment by SickClang** [comment]
> Hi, have you tried reinstalling? It fixed an issue for me.

Yes, but now it shows me only the phrase "the browser is not supported"
